<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>539</width>
    <height>496</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Добавление задачи для бэкапа</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>471</width>
     <height>381</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Имя задачи:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="taskNameEdit">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>10</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Исходный путь:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="sourcePathEdit">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>40</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="sourcePathButton">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>40</y>
      <width>131</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Выберите папку</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="destPathEdit">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>80</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Конечный путь:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="destPathButton">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>80</y>
      <width>131</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Выберите папку</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Тип бэкапа:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="backupTypeCombo">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>130</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Полный </string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Инкрементальный</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Диференциальный</string>
     </property>
    </item>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>381</width>
      <height>171</height>
     </rect>
    </property>
    <property name="title">
     <string>Время бэкапа</string>
    </property>
    <widget class="QRadioButton" name="minutesRadio">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>95</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Минуты</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="hoursRadio">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>95</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Час</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="daysRadio">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>95</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>День</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="weeksRadio">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>95</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Неделя</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="timeSpinBox">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>40</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>430</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="okButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>ОК</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cancelButton">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>0</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Отмена</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
